### ISSUE TITLE
Configuration file is blank

### ISSUE THREAD

**Issue by returnzork** [Vexatos/AsieLib#6]
Issue: Configuration file is blank, which means all values are their defaults.

**Comment by returnzork** [comment]
OLD ISSUE, SOLVED: After installing this, chat no longer shows in the server console
Using minecraft version 1.7.2, forge version 10.12.2.1133, asielib 0.2.4

this also occurs with asielib version 0.2.3

If any additional information is needed, let me know.

On a side note: the asielib.cfg file is blank

Edit: 
Looking at  ChatHandler.java, it would appear this is most likely because the config file is blank.
With chatEvent method - Appears to be sending a "direct" message to the players near you, instead of broadcasting globally?

edit 2:

could event.setCanceled(true); be causing it? 
https://github.com/asiekierka/AsieLib/blob/master/src/main/java/pl/asie/lib/chat/ChatHandler.java#L102

Edit 3: 
I looked at the config files from other mods, and was able to jerry rig the config file so it disables the chat features.
I added

base {
    B:enableChatTweaks=false
}

to the config file. Editing the issue to reflect the configuration file issue.
